src/fxa-geodb.js

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// these are defaults, can be overloaded by configuring options
+var db = __dirname + '/../db/cities-db.mmdb';
```

Comment:
`db` and `db_backup` are really `DEFAULT_DB_PATH` and `DEFAULT_BACKUP_DB_PATH` now that I see how they are used below.